### PR TITLE
Main submit script configured for def case

### PR DESCRIPTION
**Pull request by georgia14**

### CODE REVIEW

File: scripts/submit_jobs.py [hunk 1]
```diff
@@ -80,7 +83,8 @@ def generate_ntuple_config(configtype, newtag):
     sys.exit("Only running on a single run at a time is supported.")
 RUN = GOOD_RUN_DATA.keys()[0]
 # generate configs both for default and new conditions
-for jobtype in ['def', 'new_cond']:
+#for jobtype in ['def', 'new_cond']:
+for jobtype in COND_LIST:
```
Comment (christopheralanwest) [inline scripts/submit_jobs.py:87]: `COND_LIST` is never changed from its default value prior to this line. Do you intend to append 'new_cond' to `COND_LIST` if the `-t` option is specified?

Comment (christopheralanwest) [reply scripts/submit_jobs.py:87]: Could you please follow up on this comment?

Comment (georgia14) [reply scripts/submit_jobs.py:87]: Ehm sorry for this delay. Indeed, that's a good idea -- we could add that after line #109?
Please go ahead if you are ready, or I can add that tomorrow morning.
Thanks, Georgia

Comment (christopheralanwest) [reply scripts/submit_jobs.py:87]: Yes, after line #109 would work. Please make the change yourself.

**Pull request closed without merging**